tools/build-vs2015-x64.cmd

```diff
@@ -0,0 +1,6 @@
+set "VS_TOOLS_VERSION=vs2015"
+REM Ref. https://cmake.org/cmake/help/latest/generator/Visual%20Studio%2014%202015.html
+set "CMAKE_GEN=Visual Studio 14 2015 Win64"
+cd %~dp0
+call setup-buildtools.cmd
+call build.cmd -DWITH_ABSEIL:BOOL=ON
```

Comment:
I wasn't sure what you meant originally around having multiple builds.

I thought you were implying we'd have a matrix of builds

Compiler x ABSEIL_FLAG x STD_FLAG

It looks like, instead, each version of VisualStudio gets a configuration.    Do we need to make a visual matrix of that for users?  While I definitely want the ability to statically link against STD we should make sure it's obvious which configurations support which and what you get out of the box.

Reply:
Basically if we want to produce a static lib linkable with anything vs2015|vs2017|vs2019, then it MUST be built using vs2015. This is quite silly linker limitation. For example, if we statically produce a lib with vs2019, then it just won't link with older compilers!!! That's because the latest vs2019 has a bit different exception handling semantics.

I do want to have a matrix doc created.